riscv/src/continuations/bootloader.rs

```diff
@@ -52,20 +52,13 @@ pub fn bootloader_preamble() -> String {
     let mut preamble = r#"
     // ============== bootloader-specific instructions =======================
     // Write-once memory
-    let BOOTLOADER_INPUT_ADDRESS: col = |i| i;
-    let bootloader_input_value;
-    // Loads a value. If the cell is empty, the prover can choose a value.
-    instr load_bootloader_input X -> Y { {X, Y} in {BOOTLOADER_INPUT_ADDRESS, bootloader_input_value} }
-    instr assert_bootloader_input X, Y { {X, Y} in {BOOTLOADER_INPUT_ADDRESS, bootloader_input_value} }
-
-    let tmp_bootloader_value;
-
-    // Sets the PC to the bootloader input at the provided index if it is nonzero
```

Comment:
Removed "if it is nonzero". As far as I can see, this was not the case before and it is not now.